packages/react-native-sdk/src/theme/theme.ts

```diff
@@ -335,6 +361,17 @@ export const defaultTheme: Theme = {
       fontWeight: '400',
     },
   },
+  defaults: {
+    color: colors.primary,
+    backgroundColor: colors.background2,
+    margin: 10,
+    padding: 10,
+    fontSize: 16,
```

Comment:
Please do make sure that all margins, padding, sizes also move to theme when we move to design v2